web/src/services/event.ts

```diff
@@ -1,13 +1,13 @@
 import ApiService from "./api";
-import moment from 'moment';
+import moment from "moment";
 
-const EventService = {
+export class EventService {
   async getEvents() {
     const api = new ApiService();
-
     let events;
+
     try {
-      events = await api.get("events");
+      events = await api.get(`events`);
```

Comment:
Why the backtick instead of double quotes? 🤔

Reply:
Meh, remainers of something I tried before, I'll fix it!